{"name": "test_user_login_wrong", "status": "passed", "attachments": [{"name": "stdout", "source": "ee45ba23-0839-4e92-adf8-997994766c00-attachment.txt", "type": "text/plain"}], "start": 1600238213635, "stop": 1600238213901, "uuid": "c39bd66d-809f-4180-a51e-7bd66635ca77", "historyId": "8923e36b93bfcbf2b1b29bba9c76fef6", "testCaseId": "4b6bcd6bc9f4b93fe085c2939884ea45", "fullName": "testcase.test_login.TestLogin#test_user_login_wrong", "labels": [{"name": "severity", "value": "critical"}, {"name": "story", "value": "测试登录密码错误"}, {"name": "feature", "value": "测试登录模块"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "TX5C3S1L7D31UYD"}, {"name": "thread", "value": "3768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}], "links": [{"type": "test_case", "url": "/basic/auth/login", "name": "/basic/auth/login"}]}